relay-server/src/actors/global_config.rs

```diff
@@ -253,25 +257,47 @@ impl Service for GlobalConfigService {
         tokio::spawn(async move {
             let mut shutdown_handle = Controller::shutdown_handle();
 
-            relay_log::info!("global config service starting");
+            let global_config = match GlobalConfig::load(self.config.path()) {
```

Comment:
We only need this config in non-managed modes, so let's move this block of code to the section below in line 284.

Reply:
hm I did use it to notify the user that if they had the file in their config folder that it'd be ignored, but I suppose that's not really important, i moved the code